services/docker-compose.yml

```diff
@@ -1030,7 +1030,7 @@ services:
         "--loglevel",
         "verbose",
         "--databases",
-        "7",
+        "8",
```

Comment:
see my comment above, w.r.t. the needed change in redis-commander :--)

Reply:
OK the PR should be all set in OPS https://github.com/ITISFoundation/osparc-ops-environments/pull/501

Reply:
also the bump from 7 to 8 should only be required here